CONTRIBUTING.md

```diff
@@ -194,10 +201,57 @@ $ git push origin my-branch
 Go to https://github.com/yourusername/node and select your branch.
 Click the 'Pull Request' button and fill out the form.
 
-Pull requests are usually reviewed within a few days. If there are comments
-to address, apply your changes in a separate commit and push that to your
-branch. Post a comment in the pull request afterwards; GitHub does
-not send out notifications when you add commits.
+Pull requests are usually reviewed within a few days.
+
+### Step 7: Discuss and update
+
+You will probably get feedback or requests for changes to your Pull Request,
```

Comment:
Nit: period/full stop instead of comma after `Pull Request` (and, therefore, `this` -> `This` in the next line).